### ISSUE TITLE
Create and publish official Grafana Dashboard

### ISSUE THREAD

**Issue by Dentrax** [kubernetes-sigs/descheduler#1046]
**Is your feature request related to a problem? Please describe.**
People may be taking the time to make their own dashboards. (at least we did last year). We can provide an out-of-box Grafana Dashboard.

**Describe the solution you'd like**
Create and maintain the official Deschduler Grafana dashboard in this repo and publish it to [Grafana Dashboard Marketplace](https://grafana.com/grafana/dashboards)

Add the Dashboard link in the README.

**Describe alternatives you've considered**
`-`

**Additional context**
`-`

**Comment by WDaan** [comment]
Yes please!

**Comment by samavasi** [comment]
This would be great to have a starting point , is there any sample anyone can share ?

**Comment by aslafy-z** [comment]
Here's a very rough single-panel descheduler Grafana dashboard: https://gist.githubusercontent.com/aslafy-z/1a7f8cde84285bf578c9146dd95a9dcf/raw/eb50a0a0756de9076b97d0267322c87b170de51f/descheduler-grafana-dashboard.json

**Comment by mrclrchtr** [comment]
> Here's a very rough single-panel descheduler Grafana dashboard: https://gist.githubusercontent.com/aslafy-z/1a7f8cde84285bf578c9146dd95a9dcf/raw/eb50a0a0756de9076b97d0267322c87b170de51f/descheduler-grafana-dashboard.json

Hi @aslafy-z, is this dashboard working for you?

**Comment by aslafy-z** [comment]
@mrclrchtr I updated a the query with a missing field, however, it's just a filter. It works for me for version v0.26.0.

**Comment by mrclrchtr** [comment]
Thank you. I have tried it and unfortunately it only works for me if you explicitly select a strategy.
If I deselect the filter, it shows nothing.

After changing the query to: `sum(descheduler_pods_evicted{namespace=~"$namespace",strategy=~"$strategy"}) by (namespace,strategy,result)` it's always showing some data.

**Comment by aslafy-z** [comment]
You're right, the query was missing a ~ to support multiple strategies. It's now fixed.

**Comment by aslafy-z** [comment]
Not stale
